exercise10.py

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May 10 15:34:22 2018
+
+@author: amyhaddad
+"""
+
```

Comment:
@paulghaddad I used all lambda functions here. Is this the best way to go about it? In my first iteration, I used all functions, but refactored it to all lambda's. Thoughts?